src/Stripe.net.Tests/countryspecs/when_listing_country_specs.cs

```diff
@@ -17,7 +17,7 @@ public class when_listing_country_specs
         Because of = () =>
         {
             _countrySpecList = _countrySpecService.List(
-                new StripeListOptions() { Limit = 8 }
+                new CountrySpecListOptions() { Limit = 8 }
```

Comment:
why not `StripeCountrySpecListOptions`?

Reply:
To maintain consistency with all the other classes (service and entities). I agree that we should fix this, but it's a job for a different PR and in the meantime I'd rather maintain consistency within each resource.